groups/instance/profiles/shared-services-eu-west-2/devops/vars

```diff
@@ -0,0 +1,6 @@
+account_name         = "shared-services"
+environment          = "devops"
+region               = "eu-west-2"
+secrets_account_name = "development"
+secrets_environment  = "devops1"
+ssl_certificate_name = "*.companieshouse.gov.uk"
```

Comment:
I'd be tempted to move the cert name to vault. Also, add a new line or update your IDE to automatically insert new lines

Reply:
Implemented via commit https://github.com/companieshouse/artifactory-terraform/pull/19/commits/9cefa06abb1be123499b3476f7caca8d63a6cb38